corehq/apps/campaign/tests/test_models.py

```diff
@@ -16,78 +16,76 @@ def dashboard_fixture():
-    ]
+    DashboardMap.objects.create(
+        dashboard=dashboard,
+        title='Map 1',
+        case_type='type1',
+        geo_case_property='property1',
+        dashboard_tab=DashboardTab.CASES,
+        display_order=1,
+    )
+    DashboardMap.objects.create(
+        dashboard=dashboard,
+        title='Map 2',
+        case_type='type2',
+        geo_case_property='property2',
+        dashboard_tab=DashboardTab.CASES,
+        display_order=2,
+    )
+    DashboardMap.objects.create(
+        dashboard=dashboard,
+        title='Map 3',
+        case_type='type3',
+        geo_case_property='property3',
+        dashboard_tab=DashboardTab.MOBILE_WORKERS,
+        display_order=1,
+    )
+    DashboardMap.objects.create(
+        dashboard=dashboard,
+        title='Map 4',
+        case_type='type4',
+        geo_case_property='property4',
+        dashboard_tab=DashboardTab.MOBILE_WORKERS,
+        display_order=2,
+    )
+    yield
 
 
 @fixture
 def dashboard_reports():
     dashboard = dashboard_fixture()
-    yield [
-        DashboardReport.objects.create(
-            dashboard=dashboard,
-            title='Report 1',
-            report_configuration_id='report1',
-            dashboard_tab=DashboardTab.CASES,
-            display_order=1,
-        ),
-        DashboardReport.objects.create(
-            dashboard=dashboard,
-            title='Report 2',
-            report_configuration_id='report2',
-            dashboard_tab=DashboardTab.CASES,
-            display_order=2,
-        ),
-        DashboardReport.objects.create(
-            dashboard=dashboard,
-            title='Report 3',
-            report_configuration_id='report3',
-            dashboard_tab=DashboardTab.MOBILE_WORKERS,
-            display_order=1,
-        ),
-        DashboardReport.objects.create(
-            dashboard=dashboard,
-            title='Report 4',
-            report_configuration_id='report4',
-            dashboard_tab=DashboardTab.MOBILE_WORKERS,
-            display_order=2,
-        ),
-    ]
+    DashboardReport.objects.create(
+        dashboard=dashboard,
+        title='Report 1',
+        report_configuration_id='report1',
+        dashboard_tab=DashboardTab.CASES,
+        display_order=1,
+    )
+    DashboardReport.objects.create(
+        dashboard=dashboard,
+        title='Report 2',
+        report_configuration_id='report2',
+        dashboard_tab=DashboardTab.CASES,
+        display_order=2,
+    )
+    DashboardReport.objects.create(
+        dashboard=dashboard,
+        title='Report 3',
+        report_configuration_id='report3',
+        dashboard_tab=DashboardTab.MOBILE_WORKERS,
+        display_order=1,
+    )
+    DashboardReport.objects.create(
+        dashboard=dashboard,
+        title='Report 4',
+        report_configuration_id='report4',
+        dashboard_tab=DashboardTab.MOBILE_WORKERS,
+        display_order=2,
+    )
+    yield
 
 
-@use(dashboard_maps)
+@use('db', dashboard_maps)
```

Comment:
This isn't necessary locally, but on GitHub tests fail with
```
ERROR corehq/apps/campaign/tests/test_models.py::test_dashboard_map_ordering - UserWarning: non-database test corehq/apps/campaign/tests/test_models.py::test_dashboard_map_ordering triggered database setup
```

Reply:
I think this could be resolved (and `'db'` removed here) if `@use(dashboard_fixture)` was added to the `dashboard_maps` and `dashboard_reports` fixtures.